_sass/layout.scss

```diff
@@ -66,6 +66,7 @@
     padding-bottom: $gutter-spacing;
     padding-left: $gutter-spacing;
     margin-left: $nav-width;
+    margin-left: $nav-width;
```

Comment:
Is `margin-left` duplicated intentionally?

Reply:
No. I'm not sure why that happened. I'll remove it. Good catch.